.gitpod.yml

```diff
@@ -13,12 +13,31 @@ github:
 
 tasks:
   - name: prebuild
-    init: ./scripts/start.sh && ./scripts/stop.sh && gp sync-done prebuild
-    command: curl -L --http1.1 https://cnfl.io/ccloud-cli | sudo sh -s -- -b /usr/local/bin; exit
+    init: |
+      cd .. && git clone https://github.com/confluentinc/jmx-monitoring-stacks && cd cp-demo/. && \
+        ./scripts/start.sh && ./scripts/stop.sh && \
```

Comment:
@vdesabou @chuck-confluent any reason to start/stop the whole environment before starting?

Reply:
The `init` is done as part of https://www.gitpod.io/docs/prebuilds, so it doesn't really matter if it takes more time

Doing start/stop is the easiest way to pull and build connect image + generating certificates

Reply:
got it, thanks @vdesabou

Reply:
Thanks @vdesabou, that make sense.

So, every new environment will run this as the workspace is created (~15 min):
![image](https://user-images.githubusercontent.com/6180701/140044956-49700a8f-669b-4bae-b203-ce12a0c1b7ed.png)

As start.sh does much more than pulling/building images (e.g. lot of waiting, deployment), having an `init.sh` that _just_ pull/build images and generate certs could improve user experience (maybe reduce it to ~5min). wdyt? If worth, I can create another PR for it.

Reply:
aha! So, I might be hit by this: 

```
    # enable for pull requests coming from forks (defaults to false)
    pullRequestsFromForks: true
```

Should we enable that one for forks as well? Seems we are enabling on most cases anyway.

Reply:
Yes I wasn't sure to put it or not, but I think it makes sense to allow it

Reply:
adding it